pkgs/development/python-modules/pyzstd/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "pyzstd";
+  version = "0.16.0";
+
+  format = "pyproject";
```

```suggestion
  pyproject = true;
```
please replace all occurrences of this